src/ui/event-description/infos.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import t from 'prop-types'
+import styled from 'styled-components'
+import formatDate from '../../utils/formatDate'
+
+const Infos = ({ date, address }) => (
+  <InfosBox>
+    <div>{formatDate(date)}</div>
```

Comment:
Como o `formatDate` é usado em dois lugares, e serve apenas para exibir uma data no componente, ele poderia ser um componente `<FormattedDate date={date} />`, que ae usaríamos a função só dentro dele, e nem precisaria extrair ela pra `utils` (só se for precisar dela em outro lugar que não seja no componente em si)

Reply:
Obrigado pelo review ❤️
Pode deixar que vou aplicar estas sugestões maravilhosas  =)